code/modules/projectiles/projectile/energy/stun.dm

```diff
@@ -395,7 +403,12 @@
 
 // movespeed mods
 /datum/movespeed_modifier/tasing_someone
-	multiplicative_slowdown = 2
+	multiplicative_slowdown = 3
```

Comment:
can be overriden in a modular folder, and so can /obj/projectile/energy/electrode/sec